app/src/main/proxy.ts

```diff
@@ -119,6 +121,8 @@ export async function proxyStreamRequestInner(
   offset?: number,
 ): Promise<ProxyResponse> {
   return new Promise((resolve, reject) => {
+    // The path in `command.command` is hardcoded and not user input
```

Comment:
I guess we could define the type `ProxyCommand` to take `ProxyCommand.command` as predefined literals rather than a string?

Reply:
hm, actually I guess we can't quite hardcode this since we need the path to the proxy binary that we get from the vite config. never mind!!